src/main/resources/locales/en/system-admin-dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "System Admin dashboard",
+  "heading": "System Dashboard",
```

Comment:
Shall we name he heading System Admin Dashboard too? System Dashboard sounds like a dashboard for the entire system and not a dashboard for the admin to use :)

Reply:
Updated :)